2020/criteria.adoc

```diff
@@ -0,0 +1,318 @@
+= bewertung
+
+== basis 
+
+=== https://de.wikipedia.org/wiki/Digitale_Nachhaltigkeit[Voraussetzungen der digitalen Nachhaltigkeit]
+
+=== https://de.wikipedia.org/wiki/Vergleichender_Warentest#Darstellung_der_Testergebnisse[Vergleichender Warentest - Darstellung der Testergebnisse]
+
+== eigenschaften, kriterien & gewichtung
+
+[cols=">,1,1,1,1", options="header"] 
+|===
+| Projekt 
+| Gewichtung
+| Projektname 1 
+| Projektname 2
+| Projektname 3
+
+| Version / Revision
+| --
+| 
+| 
+| 
+
+| URL
+| --
+| 
+| 
+|  
+
+| 
+| 
+| 
+| 
+| 
+
+| image:https://upload.wikimedia.org/wikipedia/commons/8/89/Icon_DINA_Voraussetzungen_Digitale_Nachhaltigkeit_01_Ausgereift_Farbig.svg[Ausgereift,64]
+| 8% - Ausgereift
+| 
+| 
+| 
+
+| Qualität
+| 2%
+| 
+| 
+| 
+
+| Korrektheit
+| 2%
+| 
+| 
+| 
+
+| Sicherheit
+| 2%
+| 
+| 
+| 
+
+| Vollständigkeit
+| 2%
+| 
+| 
+| 
+
+| image:https://upload.wikimedia.org/wikipedia/commons/7/79/Icon_DINA_Voraussetzungen_Digitale_Nachhaltigkeit_02_Transparente_Strukturen_Farbig.svg[Transparente Strukturen,64]
+| 8% - Transparente Strukturen
+| 
+| 
+| 
+
+| Quellcode vollständig offengelegt
+| 4%
+| 
+| 
+| 
+
+| Datenformate nutzen offenene Standards
+| 4%
+| 
+| 
+| 
+
+| image:https://upload.wikimedia.org/wikipedia/commons/f/ff/Icon_DINA_Voraussetzungen_Digitale_Nachhaltigkeit_03_Semantische_Daten_Farbig.svg[Semantische Daten,64]
+| 8% - Semantische Daten
+| 
+| 
+| 
+
+| Daten sind untereinander verknüpft
+| 2%
+| 
+| 
+| 
+
+| Daten sind mit externen, offenen Daten verknüpft
+| 2%
+| 
+| 
+| 
+
+| Metadaten / -formate sind verfügbar
+| 2%
+| 
+| 
+| 
+
+| Daten werden bereits maschinell weiterverarbeitet, aggregiert und interpretiert
+| 2%
+| 
+| 
+| 
+
+| image:https://upload.wikimedia.org/wikipedia/commons/5/51/Icon_DINA_Voraussetzungen_Digitale_Nachhaltigkeit_04_Verteilte_Standorte_Farbig.svg[Verteilte Standorte,64]
+| 8% - Verteilte Standorte
+| 
+| 
+| 
+
+| Verteilter Betrieb an unterschiedlichen Orten
+| 4%
+| 
+| 
+| 
+
+| Redundante Datenhaltung
+| 4%
+| 
+| 
+| 
+
+| **Eigenschaften des digitalen Guts** 
+| 32% (insgesamt)
+| 
+| 
+| 
+
+| 
+| 
+| 
+| 
+| 
+
+| image:https://upload.wikimedia.org/wikipedia/commons/b/b1/Icon_DINA_Voraussetzungen_Digitale_Nachhaltigkeit_05_Freie_Lizenz_Farbig.svg[Freie Lizenz,64]
+| 8% - Freie Lizenz
+| 
+| 
+| 
+
+| Nutzung von anerkannten Open Source, Open Data oder Open Access Lizenzen footnote:osi-license[link:https://opensource.org/licenses[Liste von, vom OSI auf opensourceheit geprüften Open Source Lizenzen]]
+| 4%
+| 
+| 
+| 
+
+| Massnahmen zur Sicherstellung der Lizenzkonformität vorhanden (https://de.wikipedia.org/wiki/Contributor_License_Agreement[CLA], SCAs, ...)
```

Comment:
Ich würde hier das Contributor License Agreement (CLA) mit einem [Developer Certificate of Origin (DCO)](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin) ersetzen.

In meiner Erfahrung sind CLAs zwar grundsätzlich OK, gerade in nicht besonders Open Source affinen Firmen waren sie jedoch für mich immer wieder ein Hindernis um contributing zu dürfen. Unser interes Legal hat uns selten erbaut corporate CLAs zu unterschreiben.

Das DCO konzept kommt von der Linux Foundation und wird z.Bsp. von riesigen open source Projekten wie Kubernetes verwendet.

Generell empfinde ich beide Konzepte eher als "auf das U.S. Rechtssystem ausgerichtet". Deswegen finde ich auch gut, dass es in diesem Punkt noch ein "..." hat und er nur mit 4% bewertet wird.

Was ist mit "SCA" gemeint?

Reply:
SCA: Software-Composition-Analysis: also das aktive Prüfen von transitiven Abhängigkeitsketten und deren Implikationen

Reply:
Ich habe DCO an erster Stelle mit aufgenommen - Danke!